source/app/pages/select.js

```diff
@@ -4,15 +4,46 @@ import { connect } from 'react-redux'
 
 import FileUpload from '../components/FileUpload/FileUpload'
 import Button from '../components/Button/Button'
+import getConfig from 'next/config'
 
 import css from './select.scss'
+import Modal from '../components/Modal/Modal'
+import ModalContext from '../components/ModalContext/ModalContext'
 import SampleCollections from '../components/SampleCollections/SampleCollections'
 
 Select.propTypes = {
   dispatch: PropTypes.func,
 }
-
+const {
+  publicRuntimeConfig:{
+    DEMOMODE
+  }
+} = getConfig();
 function Select({ dispatch }) {
+  if(DEMOMODE === 'true'){
+    const [modal, setModal] = React.useState(false)
+    return (
+      <ModalContext.Provider value={{modal:modal, setModal:setModal}}>
+        <Modal onClose={() => setModal(false)} show={modal}>
+            For security reasons, uploading your own documents is a feature not enabled in this version of DUS. If you'd like to 
```

Comment:
Lets avoid mentioning security or other reasons. Can we rephrase this to-

This DUS instance is running in Read-Only Mode which does not support uploading your own documents.

To use all the features of DUS, please deploy your own instance following the instructions here